pyproject.toml

```diff
@@ -30,6 +30,12 @@ ruff = "^0.0.263 || ^0.0.264 || ^0.0.265 || ^0.0.269 || ^0.0.270"
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.poetry.scripts]
+countme-totals = "mirrors_countme.scripts.countme_totals:cli"
+countme-trim-raw = "mirrors_countme.scripts.countme_trim_raw:cli"
+parse-access-log = "mirrors_countme.scripts.parse_access_log:cli"
```

Comment:
Hmm, maybe we should name this command and module consistently with the rest if it’s going to be installed in the PATH, for instance `countme-parse-access-log.py`. What do you think?

Reply:
Sounds good. Or what about to make a command `countme` and subcommands `parse-access-log`, `totals` and `trim-raw`?
To be called like `countme parse-access-log <file>` or `countme totals -h`?

Reply:
Sounds sensible, but let’s do this separately?